draft-ietf-privacypass-architecture.md

```diff
@@ -107,162 +82,251 @@ of all participating entities.
 
 # Introduction
 
-The Privacy Pass protocol provides an anonymity-preserving mechanism for
-authorization of clients with servers. The protocol is detailed in
-{{?I-D.ietf-privacypass-protocol}} and is intended for use in the
-application-layer.
-
-The way that the ecosystem around the protocol is set up can have
-significant impacts on the stated privacy and security guarantees of the
-protocol. For instance, the number of servers issuing Privacy Pass
-tokens, along with the number of registered clients, determines the
-anonymity set of each individual client. Moreover, this can be
-influenced by other factors, such as: the key rotation policy used by
-each server; and, the number of supported ciphersuites. There are also
-client behavior patterns that can reduce the effective security of the
-server.
-
-In this document, we will provide a structural framework for building
-the ecosystem around the Privacy Pass protocol. The core of the document
-also includes policies for the following considerations.
-
-- How server key material should be managed and accessed.
-- Compatible server issuance and redemption running modes and associated
-  expectations.
-- How clients should evaluate server trust relationships.
-- Security and privacy properties of the protocol.
-- A concrete assessment and parametrization of the privacy budget
-  associated with different settings of the above policies.
-- The incorporation of potential extensions into the wider ecosystem.
-
-Finally, we will discuss existing applications that make use of the
-Privacy Pass protocol, and highlight how these may fit with the proposed
-framework.
-
-# Terminology
-
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
-"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
-document are to be interpreted as described in {{RFC2119}}.
+Privacy Pass is a protocol for authorization based on anonymous-credential
+authentication mechanisms. Typical approaches for authorizing clients,
+such as through the use of long-term cookies, are not privacy-friendly
+since they allow servers to track clients across sessions and interactions.
+Privacy Pass takes a different approach: instead of presenting linkable
+state carrying information to servers, e.g., whether or not the client
+is an authorized user or has completed some prior challenge, clients
+present unlinkable proofs that attest to this information.
+
+The most basic Privacy Pass protocol provides a set of cross-origin
```

Comment:
cross-origin is a little odd now, since the per-origin tokens aren't quite "cross-origin" in the traditional sense versus cross between Origin and Issuer.

Reply:
Hmm, I don't follow. Can you elaborate?